### DISCUSSION TITLE
Getting an error with async / await

### DISCUSSION THREAD

**Discussion by arotabi** [python-telegram-bot/python-telegram-bot#3014] [Q&A] [answered]
Hi,

I'm a total rookie with the Telegram API so was planning on going through the Intro to the API sample codes.
I'm falling at the first hurdle so some help would be appreciated.

When I run the sample code:

import asyncio
import telegram

async def main():
    bot = telegram.Bot("TOKEN") # with my valid token
    async with bot:
        print(await bot.get_me())

if __name__ == '__main__':
    asyncio.run(main())

I get the following error:

(Test1) D:\Projects\Python\Alert>python test2.py
Traceback (most recent call last):
  File "test2.py", line 15, in <module>
    asyncio.run(main())
  File "C:\Users\arota\.conda\envs\Test1\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\arota\.conda\envs\Test1\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "test2.py", line 10, in main
    async with bot:
AttributeError: __aexit__

Could someone help me with where I'm going wrong.
Cheers.

**Comment by Poolitzer** [comment, marked as answer]
You didn't install the pre version of our library, which you can do with `pip install -U python-telegram-bot --pre`.

**Reply by arotabi** [reply]
Thank you!
